nixos/modules/services/databases/postgresql.nix

```diff
@@ -622,6 +622,41 @@ in
             TimeoutSec = 120;
 
             ExecStart = "${postgresql}/bin/postgres";
+
+            # Hardening
+            CapabilityBoundingSet = [ "" ];
+            DevicePolicy = "closed";
+            PrivateTmp = false; #breaks wal-receiver test
```

Comment:
Will try to look into this.

Reply:
We could reach a shared privated /tmp through `JoinsNamespaceOf=` fwiw.

Reply:
I don't think so, the temp dir gets recreated on stop/start which is done in the test.

Reply:
Pushed a fix.